cmd/ucpd/cmd/root.go

```diff
@@ -26,7 +26,7 @@ import (
 	runtimelog "sigs.k8s.io/controller-runtime/pkg/log"
 
 	"github.com/radius-project/radius/pkg/armrpc/hostoptions"
-	"github.com/radius-project/radius/pkg/ucp/dataprovider"
+	"github.com/radius-project/radius/pkg/ucp/databaseprovider"
```

Comment:
Do we have both **database** and **databaseprovider** folders under **ucp**? I think this is the best practice, right? I am asking because I would create **database** under **ucp** and then **provider** under **database**.

Reply:
I'm planning to rearrange these in the next PR 👍 

I think it's a big improvement to name the package `databaseprovider` instead of `provider`. You'll notice that everywhere we reference these, they get aliased to something else.